build/scripts/build.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+function parse_parameters
+{
+	#Parse input parameters
+	while getopts "hr:s:c:p:d:v:n:" OPTION 
+	do
+		case $OPTION in
+		r)
+			ROOT_PATH=$OPTARG
+			;;
+		
+		s)
+			VS_SOLUTION=$OPTARG
+			;;
+			
+		c)
+			CONFIGS=$OPTARG
+			;;
+		
+		p)
+			PROJECTS=$OPTARG
+			;;
+			
+		d)
+			DEVENV_PATH=$OPTARG
+			;;
+
+		v)
+			CONAN_PACKAGE_VERSION=$OPTARG
+			;;
+
+		n)
+			CONAN_PACKAGE_CHANNEL=$OPTARG
+			;;
+			
+		h)
+			usage #help: show usage section
+			exit
+			;;	
+			
+		?)		
+			echo "Invalid parameter" 
+			usage
+			exit
+			;;
+		esac
+	done
+}
+
+
+function init_parameters
+{
+	CONAN_PACKAGE_VERSION="0.0.0.0"
```

Comment:
0.0.0 is the default we are using (3 nums considering we have channel)

Reply:
Fixed